randovania/games/fusion/logic_database/Sector 1 SRX.txt

```diff
@@ -1432,12 +1440,12 @@ Extra - room_id: [40]
               # Vanilla Method - Shinespark from Hornoad Housing -> Access
               Speed Booster and Disabled Door Lock Rando and Disabled Entrance Rando and Can Kill Eye Door
               Any of the following:
-                  # Pre-Fight, with Hi-Jump
+                  # Pre-Fight, with Hi-Jump: https://youtu.be/q7HKdMt4v3g
                   Hi-Jump and Shinespark Tricks (Intermediate)
-                  # After Fight, Trickless
+                  # After Fight, Trickless: https://youtu.be/Z6_miH4qoQY
                   Hi-Jump and After Boss Charge Core-X Defeated
-                  # No Hi-Jump
-                  Shinespark Tricks (Advanced)
+                  # After Fight, No Hi-Jump: https://youtu.be/U1nM5zVh_N0
```

Comment:
why has this been chagned to after fight? is pre-fight NHJ not possible?

Reply:
it is possible, not sure why it isn't listed here. that might've been my fault,
Not sure what difficulty it should be at tho, Pre fight NHJ can't be advanced atm due to it being logically redundant with Post fight NHJ, but Pre NHJ doesn't really scream Expert to me tbh.
You could instead maybe bump post NHJ down to intermediate and Pre w/ HJ down to beginner, that way you could have pre NHJ be at advanced, but idk if that totally fitting either.

Reply:
the reason why it isnt listed is because clue changed it to include `After Fight`

Reply:
i mean, yeah both both pre fight and post fight NHJ shouldve been on here in the first place

Reply:
They were.
`Shinespark Tricks (Advanced)` implies 
1. No HiJump
2. Post OR Pre fight

Reply:
I changed it mostly to match the video upload. There wasn't a separation that I saw in main. I can adjust it back. Though the trick itself is basically the same otherwise.

Reply:
> They were. `Shinespark Tricks (Advanced)` implies
> 
>     1. No HiJump
> 
>     2. Post OR Pre fight

I guess that makes sense but imo, the softlock potential makes me want to differentiate the two

Reply:
I've added a 4th section for: Pre-fight, no high jump.